helm/sealed-secrets/templates/role-binding.yaml

```diff
@@ -38,11 +40,16 @@ roleRef:
   kind: Role
   name: {{ printf "%s-service-proxier" (include "sealed-secrets.fullname" .) }}
 subjects:
-- apiGroup: rbac.authorization.k8s.io
-  kind: Group
-  name: system:authenticated
-{{ end }}
+  {{- range .Values.rbac.proxier.subjects }}
```

Comment:
I would use the `sealed-secrets.render` function to render the subjects. Like this:
```
{{- include "sealed-secrets.render" (dict "value" .Values.rbac.proxier.subjects "context" $) | nindent 2 }}
```